composer.json

```diff
@@ -10,14 +10,17 @@
         }
     ],
     "require": {
+        "php": "^7.1",
+        "ext-pdo": "*",
```

Comment:
Could you please move this to the `suggest` section of and add a message about why this is suggested (like for what command types)